app/router.js

```diff
@@ -16,9 +16,7 @@ Router.map(function() {
   this.route('saml2', {path: '/saml2'}, function() {
     this.route('redirect', {path: '/redirect'});
   });
-  if(ENV.isRegistrationEnabled) {
```

Comment:
@RaiVaibhav why was this check removed?

Reply:
it moved to register route file because of logic was to actually redirect if the `register` route is there with additionally where they are allowed or not